packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -241,10 +260,16 @@ class ScrollDragController implements Drag {
   @override
   void update(DragUpdateDetails details) {
     assert(details.primaryDelta != null);
+    final bool lastEventWasStationary = _lastDetails is DragUpdateDetails
+        && _lastDetails.primaryDelta == 0.0;
```

Comment:
nitty nit nit: please line up the `_lastDetails`s on both lines (probably putting the && on the previous line)